packages/kbn-shared-ux-components/src/page_template/no_data_page/action_cards/action_cards.stories.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { NoDataCard } from '../no_data_card';
+import { ActionCards } from './action_cards';
+
+export default {
+  title: 'Elastic Action Cards',
+  description: 'A group of ElasticAgentCards or NoDataCards, to be used on NoData page',
+};
+
+export const PureComponent = () => {
+  const action = {
+    recommended: false,
```

Comment:
We can just leave this off as it's a deprecated prop.
```suggestion
```

Reply:
Not sure what you mean here? `recommended` is a property for applying the beta badge... If it's not needed anymore, should we remove it from the Props interface as well?